src/MusicStore.Standalone/MusicStore.Standalone.xproj

```diff
@@ -17,5 +17,5 @@
   <PropertyGroup>
     <SchemaVersion>2.0</SchemaVersion>
   </PropertyGroup>
-  <Import Project="$(VSToolsPath)\DotNet\Microsoft.DotNet.targets" Condition="'$(VSToolsPath)' != ''" />
+  <Import Project="$(VSToolsPath)\DotNet.Web\Microsoft.DotNet.Web.targets" Condition="'$(VSToolsPath)' != ''" />
```

Comment:
Note - this is what prevented MusicStore.Standalone from being launched from VS. I changed the other file just because I was touching this.